modules/virt-about-libguestfs-tools-virtctl-guestfs.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-libguestfs-tools-virtctl-guestfs.adoc
+
+[id="virt-about-libguestfs-tools-virtctl-guestfs_{context}"]
+= About libguestfs tools and virtctl guestfs
+
+The `virtctl guestfs` command contains many useful sub-commands that are helpful for modifying, inspecting, and debugging VM disks on a PVC. To see a complete list of possible `virt` commands, run `virt-` from a command line terminal. Some examples are:
+
+* To edit a file interactively in your terminal, use `virt-edit -a /dev/vda /etc/motd`.
+* To inject an ssh key into the guest and create a login, use `virt-customize -a /dev/vda --ssh-inject root:string:<public key example>`.
+* To see how much disk space is used by a VM, use `virt-df -a /dev/vda -h`.
+* To see the full list of all RPMs installed on a guest, use `virt-customize -a /dev/vda --run-command 'rpm -qa > /rpm-list'` to create an output file containing the full list. Then use `virt-cat -a /dev/vda /rpm-list` to display the output list in your terminal.
+
+By default, `virtctl guestfs` creates a container with everything needed for running a VM. However, the command also uses several flag options if you want to customize the container:
```

Comment:
I would suggest: "By default, `virtctl guestfs` creates a session with everything needed to manage a VM disk.  However, the command also uses several flag options if you want to customize the behavior:"

Reply:
Should we mention the `-n <namespace>` option here even though it doesn't appear in the command's help string?

Reply:
Also, should probably explicitly state that the PVC name is a required argument